### DISCUSSION TITLE
MCDU Custom Font Error

### DISCUSSION THREAD

**Discussion by lamboman43** [flybywiresim/aircraft#3849] [Help] [answered]
I'm very new to GitHub and I don't know if I'm allowed to post a bug report under the Issues tab as I'm not a developer or QA tester. I noticed these few instances of font errors ever since the MCDU font was updated to the custom font.
This first image shows a rectangle where I believe the letter "N" was with the previous font. To get to this page start at the MCDU Menu > ATSU > AOC MENU > INIT/PRES and then scroll to the second page using the right-pointing MCDU scrolling arrow.

![Microsoft Flight Simulator 3_8_2021 8_03_26 AM](https://user-images.githubusercontent.com/80269628/110339235-29c5cf80-7fe5-11eb-9b89-c99de6b4b8d0.png)

For this second photo, I believe you need to have an active flight plan initialized under the MCDU INIT button. It shows up to the left of "ARRIVAL" when requesting an AOC ATIS report. It also shows up if you select DEPARTURE and ENROUTE. To get there start at the MCDU Menu > ATSU > AOC MENU > ATIS

![Microsoft Flight Simulator 3_8_2021 8_03_05 AM](https://user-images.githubusercontent.com/80269628/110339158-11ee4b80-7fe5-11eb-98ef-66e70e7fb0ee.png)

I also noticed that often these rectangles will show up when the MCDU displays the ATIS report. 

![Microsoft Flight Simulator 3_8_2021 8_17_15 AM](https://user-images.githubusercontent.com/80269628/110340703-c76dce80-7fe6-11eb-80d6-4fcddfee2723.png)

When I choose to print these ATIS reports, it looks like those rectangles are in positions where a comma ( , ) is supposed to be. 

![Microsoft Flight Simulator 3_8_2021 8_17_21 AM](https://user-images.githubusercontent.com/80269628/110340920-07cd4c80-7fe7-11eb-8beb-c822e0eff203.png)

Obviously, with the first two pictures, commas aren't supposed to be in those positions so I don't know what those rectangles are actually supposed to show, but I doubt they are meant to be there. 

P.S. If it matters, these photos were all taken from a cold and dark start.

**Comment by hotshotp** [comment, marked as answer]
@tyler58546

**Reply by tracernz** [reply]
> 
> 
> @tyler58546

https://github.com/flybywiresim/a32nx/pull/3847

**Comment by ExampleWasTaken** [comment]
To help you a bit with GitHub here: You can totally post bugs you found as issues. Just be sure to follow the correct template. 
You don't need to repost this report as an issue. Just for the future ;)

**Reply by lamboman43** [reply]
Perfect, thank you for the advice!
